lib/handler.js

```diff
@@ -68,6 +70,13 @@ const handler = async (req, res, pgPool) => {
       res,
       pgPool,
       fetchMinersRSRSummary)
+  } else if (req.method === 'GET' && segs[0] === 'nodes' && segs[1] === 'daily' && segs.length === 2) {
```

Comment:
Let's treat the paths `/nodes/daily` and `/nodes/monthly` reserved so that we can implement them to return data in the same format as other `daily` and `monthly` endpoints already do.

We have been already using the term "nodes" to describe unique pairs of `(wallet_address, inet_group)`. Let's use "stations" instead.

Few endpoint names to consider:
- `/stations`
- `/stations/raw`
- `/stations/daily/raw`
- `/stations/daily/full`
- `/stations/daily/details`

Reply:
Makes sense! Should we also rename the table and references to it in spark-evaluate (it's currently daily_node_metrics)